prow/Makefile

```diff
@@ -25,82 +26,5 @@ update-config: get-cluster-credentials
 		--plugins-config-path=plugins.yaml \
 		--wet
 
-# use to guarantee that on command completion all deployments see new configs
-update-configs: update-config update-plugins
-
-.PHONY: update-config update-plugins get-cluster-credentials update-configs
```

Comment:
Lets keep the applicable `.PHONY` targets and the `update-configs` target?
Also it looks like `update-plugins` doesn't exist and is covered by `update-config`.

Reply:
Done